modules/avoindata-drupal-theme/bootstrap/less/dropdowns.less

```diff
@@ -34,7 +34,6 @@
   left: 0;
   z-index: @zindex-dropdown;
   display: none; // none by default, but block on "open" of the menu
-  float: left;
```

Comment:
We don't want to change any upstream files. I would create a new class in template and add the float change eg. in overrides.less